src/st/class-wpml-pb-string-translation.php

```diff
@@ -27,9 +27,6 @@ public function new_translation( $translated_string_id ) {
 			$package = $this->factory->get_wpml_package( $package_id );
 			if ( $package->post_id && $this->strategy->get_package_kind() === $package->kind ) {
 				$this->add_package_to_update_list( $package, $language );
-				if ( defined( 'DOING_AJAX' ) && DOING_AJAX || defined( 'XMLRPC_REQUEST' ) && XMLRPC_REQUEST ) {
-					$this->save_translations_to_post();
-				}
```

Comment:
@strategio are you sure this condition wasn't added there for a specific reason?

Because that's what it looks like to me. It seems like we had to solve a problem (maybe the compatibility with a different page builder) so we added these lines.

Reply:
Better to ask @BruceOnTheGo @brucepearson about this.

Reply:
@andreasciamanna, @adrianosferreira, quoting myself in the YT ticket:
>Looking at the Git history in ST, I can see that this action to update the post has been moved to the shutdown in wpmlst-963. At this time, we should have removed this call in `\WPML_PB_String_Translation::new_translation` I guess.
>
>Then later, we also started to update the post when handling an XML-RPC request. Looking at the original tickets (wpmlst-1210 / wpmltm-1759), I think wpmlst-1210 was not needed (and tested this successfully this morning).